common/src/test/scala/uk/ac/wellcome/platform/stacks/common/services/CatalogueServiceTest.scala

```diff
@@ -0,0 +1,92 @@
+package uk.ac.wellcome.platform.stacks.common.services
+
+import org.scalatest.concurrent.{IntegrationPatience, ScalaFutures}
+import org.scalatest.{FunSpec, Matchers}
+import uk.ac.wellcome.platform.stacks.common.fixtures.ServicesFixture
+import uk.ac.wellcome.platform.stacks.common.models._
+
+class CatalogueServiceTest
+  extends FunSpec
+    with ServicesFixture
+    with ScalaFutures
+    with IntegrationPatience
+    with Matchers {
+
+  describe("CatalogueService") {
+    describe("getStacksWork") {
+      it("should return a StacksWork") {
+        withCatalogueService { catalogueService =>
+
+          val stacksWorkIdentifier = StacksWorkIdentifier(
+            "cnkv77md"
+          )
+
+          whenReady(
+            catalogueService.getStacksWork(stacksWorkIdentifier)
+          ) { actualWork =>
+
+            val expectedWork = StacksWork(
+              id = "cnkv77md",
+              items = List(
+                StacksItemWithOutStatus(
+                  id = StacksItemIdentifier(
+                    catalogueId = CatalogueItemIdentifier("ys3ern6x"),
+                    sierraId = SierraItemIdentifier("1292185")),
+                  location = StacksLocation("sicon", "Closed stores Iconographic")
+                )
+              )
+            )
+
+            actualWork shouldBe expectedWork
+          }
+
+        }
+      }
+    }
+
+    describe("getStacksItem") {
+      it("should get a StacksItem for a SierraItemIdentifier") {
```

Comment:
When do we lookup _from_ SierraIds?

Reply:
And should the be the duty of the Catalogue service?

Reply:
Reading through this, I think I might have misunderstood our conversation re: Catalogue vs Sierra vs Stacks.

My understanding was Sierra returns Sierra stuff, catalogue catalogue stuff, and stacks combines the two and returns stacks stuff?

e.g.
```scala
SierraService.getItem("i123456") // only utilises sierra IDs
CatalogueService.getItem("asdh673j") // only utilised catalogue IDs
StacksService.getItem("asdh673j") // interfaces with catalogue IDs as that's what we expose to clients, but can return both IDs as it knows of both
```

Reply:
The Catalogue API contains Sierra Ids & Catalogue Ids for items, and provides the basis for conversion between those identifiers. 

Specifically we look up from Sierra Ids to Catalogue Ids when converting `StacksUserHolds[SierraItemIdentifier]` to  `StacksUserHolds[CatalogueItemIdentifier]`.

TLDR: The catalogue contains sierra ids.